modules/invibesBidAdapter.js

```diff
@@ -372,7 +372,7 @@ function generateRandomId() {
 }
 
 function getDocumentLocation(topWin) {
-  return topWin.location.href.substring(0, 300).split(/[?#]/)[0];
+  return topWin.location.href.substring(0, 300);
```

Comment:
Why not use site.page on the request object

Reply:
prebid team and community have put a lot of work into making sure https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js gives the best possible results. Publishers and buyers often expect consistent determinations of site.page across integrations.

Reply:
Hello @patmmccann ! I understand the concern but you are requesting a  change in the Invibes adapter that is not part of the scope of this pull request. From my knowledge this is not standard prebid policy.

This pull request however was for a bugfix on one of our integrations, the change was minor (removal of split function, NOT access to window object) and we needed to implement it as fast as possible (it has been dragging on for weeks at this point)

We will of course implement this if required as a SEPARATE issue, with a different pull request and we require the appropriate time for it.

Thank you for understanding

Reply:
This is not out of scope; you are accessing the page url incorrectly. Your pr revises how to gather the pageurl. It remains incorrect

Reply:
Hey @patmmccann, we have made the requested changes

Reply:
Hey @patmmccann , an update please?